scene/debugger/scene_debugger.cpp

```diff
@@ -2340,11 +2341,29 @@ void RuntimeNodeSelect::_set_selection_visible(bool p_visible) {
 
 // Copied and trimmed from the CanvasItemEditor implementation.
 void RuntimeNodeSelect::_find_canvas_items_at_pos(const Point2 &p_pos, Node *p_node, Vector<SelectResult> &r_items, const Transform2D &p_parent_xform, const Transform2D &p_canvas_xform) {
-	if (!p_node || Object::cast_to<Viewport>(p_node)) {
+	if (!p_node || Object::cast_to<Window>(p_node)) {
 		return;
 	}
 
+	Transform2D xform = p_canvas_xform;
 	CanvasItem *ci = Object::cast_to<CanvasItem>(p_node);
+	CanvasLayer *cl = Object::cast_to<CanvasLayer>(p_node);
+
+	if (SubViewport *sv = Object::cast_to<SubViewport>(p_node)) {
```

Comment:
We generally discourse declaring variables in if statements, I think this hurts readability here